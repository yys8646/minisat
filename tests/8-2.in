8
.0..0...
........
1..1...1
.00..0..
....1...
.1...00.
1.......
.....1.0
