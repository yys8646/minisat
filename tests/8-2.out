8
00110011
01100110
10011001
10010011
01101100
01101001
10010110
11001100
